src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -25,10 +32,25 @@ class BehandlingService(private val behandlingRepository: BehandlingRepository,
         return opprettFørstegangsbehandling(opprettTilbakekrevingRequest)
     }
 
-    fun opprettBehandlingManuell(opprettTilbakekrevingRequest: OpprettTilbakekrevingRequest) : Behandling{
+    fun opprettBehandlingManuell(opprettTilbakekrevingRequest: OpprettTilbakekrevingRequest): Behandling {
         TODO("Ikke implementert ennå")
     }
 
+    @Transactional(readOnly = true)
+    fun hentBehandling(eksternBrukId: UUID): HentBehandlingResponsDto {
+        val behandling = behandlingRepository.findByEksternBrukId(eksternBrukId)
```

Comment:
val behandling = behandlingRepository.findByEksternBrukId(eksternBrukId) ?: throw Feil(message = "Behandling finnes ikke for eksternBrukId=$eksternBrukId",
                   frontendFeilmelding = "Behandling finnes ikke for eksternBrukId=$eksternBrukId")